core/async/Cargo.toml

```diff
@@ -23,6 +23,7 @@ tokio = { workspace = true, features = ["rt", "macros"] }
 tracing.workspace = true
 
 near-async-derive.workspace = true
+# fix_me: we use it only for logging. i think, it's a bit too much...
```

Comment:
please don't forget to add TODO(#11652) here :)

Reply:
Damn, I forgot to press Ctrl+S on that file. Haha. Thanks for the careful review!